image/build

```diff
@@ -1,12 +1,14 @@
 #!/usr/bin/env bash
 set -eu
 
-rm -rf build.tmp
-mkdir build.tmp
-(cd build.tmp
-  git clone https://github.com/intel/sriov-cni --branch v2.3 --depth 1
-  cd sriov-cni
-  make
+tmpdir="$(mktemp -d)"
+trap 'rm -rf "$tmpdir"' EXIT
+(
+ cp Dockerfile install-sriov-cni.sh "$tmpdir"
```

Comment:
FWIW, I originally tried passing in the tmpdir path to the built sriov binary as a `--build-arg` but Docker got all snitty about the path, so I had to just do everything in the tmpdir instead.